tutorials/scripting/gdextension/gdextension_file.rst

```diff
@@ -149,46 +153,54 @@ For example:
 
     GDExample = "res://icons/gd_example.svg"
 
-The path should point to a 16 by 16 pixel SVG image. Read the guide for :ref:`creating icons <doc_editor_icons>`
+The path should point to a 16×16 pixel SVG image, with two options enabled on the
+image in the Import dock:
+
+- **Editor > Scale with Editor Scale**.
+- **Editor > Convert Colors with Editor Theme**.
+
+Enabling both options ensures the icon behaves as closely as possible to
+the stock editor icons. Read the guide for :ref:`creating icons <doc_editor_icons>`
 for more information.
 
 Dependencies section
 --------------------
 
-In this section you set the paths of the GDExtension dependencies. This is used internally to export the dependencies
+In this section, you set the paths of the GDExtension dependencies. This is used internally to export the dependencies
 when exporting your game executable. You are able to set which dependency is loaded depending on the feature flags
 of the exported executable. In addition, you are able to set an optional subdirectory to move your dependencies into.
-If no path is supplied Godot will move the libraries into the same directory as your game executable.
+If no path is supplied, Godot will move the libraries into the same directory as your game executable.
 
 .. warning::
-    In MacOS it is necessary to have shared libraries inside a folder called ``Frameworks`` with a directory structure
-    like this: ``Game.app/Contents/Frameworks``.
+
+    On macOS, it is necessary to have shared libraries inside a folder called ``Frameworks``
+    with a directory structure like this: ``Game.app/Contents/Frameworks``.
 
 .. code-block:: none
 
     [dependencies]
 
     macos.debug = {
-        "res://bin/libdependency.macos.template_debug.framework" : "Contents/Frameworks"
+        "./bin/libdependency.macos.template_debug.dylib" : "Contents/Frameworks"
     }
```

Comment:
I'm not sure if I should have changed this part; let me know if I should revert that.